RHIDP-5975 document the `includeTransitiveGroupOwnership` option (#989)
redhat-developer/red-hat-developers-documentation-rhdh: modules/authentication/snip-common-authentication-options.adoc

> `includeTransitiveGroupOwnership`::

Reviewer: I think it might be better to focus on the user-facing result/actual use case of this config, which is the change in the behaviour of the `$ownerRefs` alias in RBAC. 

 Imo this would fit better under the Conditional policy aliases section [here](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.4/html/authorization/con-rbac-conditional-policies-rhdh_title-authorization#con-rbac-conditional-policies-rhdh_title-authorization).

Author: I think I see what you mean. But

* `includeTransitiveGroupOwnership` with Authentication
  * (+) User goes through all authentication options in the same place.
  * (-) Configuration item is far way from the doc explaining the consequences of the option.

* `includeTransitiveGroupOwnership` with `$ownerRefs` alias in RBAC
	* (+) `$ownerRefs` alias behavior close to how to modify it.
  * (-) the user context is using the app, no reconfiguration of the app-config file involved.

> Enable this option to add transitive parent groups into the resolved user group membership during the authentication process, therefore to include the parent group of the user's direct group in the user ownership entities.

Reviewer: ```suggestion
Enable this option to add transitive parent groups in the resolved user group membership during authentication. This ensures that a user’s indirect parent groups, in addition to their direct group, are reflected in the user ownership entities.
```

> Enable this option to add transitive parent groups in the resolved user group membership during authentication. This ensures that a user’s indirect parent groups, in addition to their direct group, are reflected in the user ownership entities.

Author: ```suggestion
Enable this option to add transitive parent groups in the resolved user group membership during authentication. This ensures that a user’s indirect parent groups, in addition to their direct group, are reflected in the user ownership entities.
See: xref:con-rbac-conditional-policies-rhdh_title-authorization[].
```

`includeTransitiveGroupOwnership`::

> Enable this option
> to add transitive parent groups into the resolved user group membership during the authentication process, therefore to include the parent group of the user's direct group in the user ownership entities.

Author: ```suggestion
Enable this option to add transitive parent groups into the resolved user group membership during the authentication process, therefore to include the parent group of the user's direct group in the user ownership entities.
```

> See: xref:con-rbac-conditional-policies-rhdh_title-authorization[].

Author: ```suggestion
See: xref:{authorization-book-url}#con-rbac-conditional-policies-rhdh_title-authorization[].
```

> See: xref:{authorization-book-url}#con-rbac-conditional-policies-rhdh_title-authorization[Conditional policies in {product}].

Author: ```suggestion
See: link:{authorization-book-url}#con-rbac-conditional-policies-rhdh_title-authorization[Conditional policies in {product}].
```
